media-sound/sox/sox-14.4.2-r2.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="The swiss army knife of sound processing programs"
+HOMEPAGE="http://sox.sourceforge.net"
+SRC_URI="mirror://sourceforge/sox/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-solaris"
+IUSE="alsa amr ao debug encode flac id3tag ladspa mad magic ogg openmp oss opus png pulseaudio sndfile static-libs twolame wavpack"
+
+BDEPDEND="virtual/pkgconfig"
+RDEPEND="
+	dev-libs/libltdl:0=
+	>=media-sound/gsm-1.0.12-r1
+	alsa? ( media-libs/alsa-lib )
+	amr? ( media-libs/opencore-amr )
+	ao? ( media-libs/libao )
+	encode? ( >=media-sound/lame-3.98.4 )
+	flac? ( >=media-libs/flac-1.1.3 )
+	id3tag? ( media-libs/libid3tag )
+	ladspa? ( media-libs/ladspa-sdk )
+	mad? ( media-libs/libmad )
+	magic? ( sys-apps/file )
+	ogg? (
+		media-libs/libogg
+		media-libs/libvorbis
+	)
+	opus? (
+		media-libs/opus
+		media-libs/opusfile
+	)
+	png? (
+		media-libs/libpng:0=
+		sys-libs/zlib
+	)
+	pulseaudio? ( media-sound/pulseaudio )
+	sndfile? ( >=media-libs/libsndfile-1.0.11 )
+	twolame? ( media-sound/twolame )
+	wavpack? ( media-sound/wavpack )"
+DEPEND="${BDEPEND}
```

Comment:
This is not needed, they're basically the same thing but BDEPEND-dependency only needs to be executed. So take BDEPEND out of here, but do keep RDEPEND.
`DEPEND="${RDEPEND}"`

Reply:
Thanks! Just poking around the portage codebase and docs, this is something that wasn't immediately clear. Just roaming the portage repo, I see that RDEPEND is usually just appended to DEPEND, and the same thing for the BDEPEND var (the examples I found while searching were just appending BDEPEND to the DEPEND list). FWIW pre-your-suggestions its not clear to me which env vars are used on their own or if everything is passed through DEPEND.